backend/src/db/entities/cropTemplate.ts

```diff
@@ -0,0 +1,34 @@
+
+import { string } from '@tensorflow/tfjs-node';
```

Comment:
I think you probably didn't mean to import this.

Reply:
removed